### PR TITLE
[customer_end_points] Initial_changes for customer endpoints

### PR DESCRIPTION
**Pull request by SiddharthaShandilya**
1. shifted databse.py file to mongodb directory
2. created config file to store the mongodb connectionurl updated main.py for 3 operation
3.  Here i am using customer email id as customer_id [ since it's easier to remember]

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import os
+
+MONGODB_CONNECTION_URL = os.environ["MONGODB_URL"]
```
Comment (abhibansal60) [inline config.py:3]: Good Move!

File: main.py [hunk 2]
```diff
@@ -1,19 +1,76 @@
+from typing import Union
```
Comment (abhibansal60) [inline main.py]: We are using python 3.11 i think, do we still need to use Union?

How about using Union operator |

File: mongodb/database.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+
+def db_connection():
+    client = AsyncIOMotorClient(config.MONGODB_CONNECTION_URL, tlsCAFile=ca)
+    db = client[config.MONGODB_DATABASE_NAME]
+    return db
+
+async def find_document(collection_name: str, query: dict, multiple: bool= False) -> UnionType[dict ,None]:
+    try:
+        if not multiple:
```
Comment (abhibansal60) [inline mongodb/database.py:19]: I liked this implementation

**Review by abhibansal60 — APPROVED** [review]
Looks good, really appreciate the clean code approach @SiddharthaShandilya

**Review by SumukhaNadig — APPROVED** [review]

File: main.py (hunk 2 above)
Comment (SiddharthaShandilya) [reply main.py]: @abhibansal60 update the code with union operator

**Review by abhibansal60 — APPROVED** [review]
Looks good

**Pull request merged**